cg/store/filters/status_metrics_filters.py

```diff
@@ -13,6 +13,12 @@ def filter_total_read_count_for_sample(metrics: Query, sample_internal_id: str,
     return total_reads_query
 
 
+def filter_above_q30_threshold(metrics: Query, q30_threshold: int, **kwargs) -> Query:
```

Comment:
Would be nice with a doc str mentioning the conversion to fraction of the q30 threshold before comparing

Reply:
I think that is an implementation detail that the caller does not need to know about, maybe it is confusing to have in the doc string?

Reply:
Leaving it as is for now